src/Vault.sol

```diff
@@ -32,48 +33,32 @@ import "./interfaces/ILiquidityPool.sol";
  */
 contract Vault {
 
+    constructor() {}
+
+    /*///////////////////////////////////////////////////////////////
+                          VAULT MANAGEMENT
+    ///////////////////////////////////////////////////////////////*/
+
     /**
      * @dev Storage slot with the address of the current implementation.
      *      This is the keccak-256 hash of "eip1967.proxy.implementation" subtracted by 1.
      */
     bytes32 internal constant _IMPLEMENTATION_SLOT =
         0x360894a13ba1a3210667c828492db98dca3e2076cc3735a920a3ca505d382bbc;
 
-    uint256 public constant yearlyBlocks = 2628000;
-
-    /*///////////////////////////////////////////////////////////////
-                INTERNAL BOOKKEEPING OF DEPOSITED ASSETS
-  ///////////////////////////////////////////////////////////////*/
-    address[] public _erc20Stored;
-    address[] public _erc721Stored;
-    address[] public _erc1155Stored;
-
-    uint256[] public _erc721TokenIds;
-    uint256[] public _erc1155TokenIds;
-
-    /*///////////////////////////////////////////////////////////////
-                          EXTERNAL CONTRACTS
-  ///////////////////////////////////////////////////////////////*/
-    address public _registryAddress; /// to be fetched somewhere else?
-    address public _liquidityPool;
-    address public _debtToken;
-
-    // ACCESS CONTROL
-    address public owner;
-    mapping(address => bool) public allowed;
-
-
     // Each vault has a certain 'life', equal to the amount of times the vault is liquidated.
     // Used by the liquidator contract for proceed claims
     uint256 public life;
-
     bool public initialized;
```

Comment:
I believe the initialized var can be removed, init is now checked with vault version.